crates/server/Cargo.toml

```diff
@@ -100,6 +100,9 @@ dns-over-h3 = [
 tls-openssl = ["dns-over-openssl"]
 tls = ["dns-over-openssl"]
 
+webpki-roots = ["hickory-resolver/webpki-roots"]
```

Comment:
So now given that `hickory-resolver` is optional here, should this be a weak dependency? (As in `hickory-resolver?/webpki-roots`.)

Reply:
Other `hickory-server` features also do not make use of weak dependency though...

Reply:
The lack of "weak dependency" usage is probably more an artifact of the age of this project rather than a strong reason one way or the other, fwiw. The goal should be to have a reliable build, so I'm generally in favor of merging this. @djc, would making this weak require other dependencies to be updated?

Reply:
I don't think so.